demo/index.html

```diff
@@ -34,11 +35,16 @@
     margin: 0;
     padding: 0;
   }
+  url-bar {
+    background-color: white;
```

Comment:
The reason this is necessary is that this demo sometimes updates the body element's background-color without also updating the body element's text color, resulting in black text on a black background in some screens.

IMO in most cases inheriting the color and background color is pretty sensible for the URL bar element.